example/javascriptlib/testing/1-test-suite/jest.config.ts

```diff
@@ -1,23 +0,0 @@
-import {pathsToModuleNameMapper} from 'ts-jest';
```

Comment:
Is there a reason these files were deleted?

Reply:
https://github.com/com-lihaoyi/mill/pull/4398#issuecomment-2614864727